multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
But... The base template is in a folder called `templates/` at the top (project) level....

Sent from Windows Mail

From: Eugene<mailto:notifications@github.com>
Sent: ?Thursday?, ?26? ?March?, ?2015 ?00?:?28
To: exclusive-gix/cs2102-store<mailto:cs2102-store@noreply.github.com>
Cc: Diga Widyaprana<mailto:a0114171@u.nus.edu>


In multimedia/templates/multimedia/base.html<https://github.com/exclusive-gix/cs2102-store/pull/16#discussion_r27138089>:

> @@ -0,0 +1 @@
> +{% extends "base.html" %}


the otherway round, we don't put our template in projects template. but rather application templates.

-
Reply to this email directly or view it on GitHub<https://github.com/exclusive-gix/cs2102-store/pull/16/files#r27138089>.